### ISSUE TITLE
Crash when resourcepack is folder not a zip

### ISSUE THREAD

**Issue by Disketaa** [Darkhax-Minecraft/Open-Loader#31]
Would be nice if I will able to set path to resources / data folder
Something like this:

```
resourcepacks = [ "config/built-in/resources/" ]
datapacks = [ "config/built-in/data/" ]
```

**Comment by Disketaa** [comment]
or can I? With` "additionalFolders": []` param

**Comment by Disketaa** [comment]
Can't figure out how to make it work, how the path should be written

**Comment by Disketaa** [comment]
https://mclo.gs/RUJDVNv

**Comment by Darkhax** [comment]
Folder packs have been fixed in 20.0.4 which is being processed by CurseForge and Modrinth right now. 

There are a few different ways to load from new folders but I would do something like this. I've tested and made sure it works.

```json
{
  "resourcePacks": {
    "enabled": true,
    "additionalFolders": []
  },
  "dataPacks": {
    "enabled": true,
    "additionalFolders": ["./config/built-in/data/"]
  }
}
```
